src/components/Alerts/AlertLabel/LabelSequence.tsx

```diff
@@ -0,0 +1,66 @@
+import ClearIcon from '@mui/icons-material/Clear';
+import LocalFireDepartmentIcon from '@mui/icons-material/LocalFireDepartment';
+import QuestionMarkIcon from '@mui/icons-material/QuestionMark';
+import { useTheme } from '@mui/material';
+import Chip from '@mui/material/Chip';
+
+import { useTranslationPrefix } from '@/utils/useTranslationPrefix';
+
+interface LabelSequenceProps {
+  isSmall?: boolean;
+  isWildfire: boolean | null;
+}
+export const LabelSequence = ({
```

Comment:
`LabelSequence` means "the sequence of the label", whereas `SequenceLabel` means "the label of the sequence" which is what we want I think

```suggestion
export const SequenceLabel = ({
```